poetry/orb.yaml

```diff
@@ -26,16 +26,24 @@ commands:
         type: string
     steps:
       - run:
-          name: install curl
+          name: Install curl
           command: |
-            if which apk >/dev/null; then
-              apk add --no-cache --no-progress curl
-            elif which apt-get >/dev/null; then
-              apt-get update -qy
-              apt-get install -qy curl
+            if cat /etc/issue | grep "Alpine" >/dev/null 2>&1; then
+              if [ "$ID" = 0 ]; then export SUDO=""; else export SUDO="sudo"; fi
```

Comment:
I was under the impression Alpine doesn't tend to ship with the `sudo` binary, do you have a sample flow for how you tested this which I could play with?

I haven't ever really had opportunity to do this, but I have vague recollections about them using `su-exec` instead?

Reply:
Looks like the default alpine doesn't ship with either, so in any case it needs to be installed beforehand.

Reply:
Hmm, is running alpine with ID!=0 just not possible, then, if you need to use non-default packages? Maybe we need to make that an error case? Or avoid setting this `SUDO` value in that case? Just want to make sure we aren't breaking any cases which previously worked, here.